project.clj

```diff
@@ -31,7 +31,8 @@
                  [org.clojure/math.combinatorics "0.1.4"]]
   :plugins [[lein-gorilla "0.4.0"]
             [lein-cljsbuild "1.1.4"]
-            [cider/cider-nrepl "0.18.0-SNAPSHOT"]]
+            [cider/cider-nrepl "0.18.0-SNAPSHOT"]
+            [lein-cloverage "1.0.13"]]
```

Comment:
Do we need to push a version cloverage in it? Surely it's just for testing coverage locally and we won't need it (for the most part after this round?)